### PR TITLE
Try fix the build

### PR DESCRIPTION
**Pull request by JayH5**

### CODE REVIEW

File: tests/test.py [hunk 1]
```diff
@@ -271,7 +271,7 @@ def test_manifest_static_storage_file(self, web_container):
         """
         hashed_svg = web_container.exec_find(
             ['static/admin/img', '-regextype', 'posix-egrep', '-regex',
-             '.*\.[a-f0-9]{12}\.svg$'])
+             r'.*\.[a-f0-9]{12}\.svg$'])
```
Comment (JayH5) [inline tests/test.py:274]: Technically this isn't a regex for Python but... I think this is better than double-escaping things?

**Comment by bearnard** [comment]
👍

**Review by jerith — APPROVED** [review]
:+1:

File: tests/test.py (hunk 1 above)
Comment (jerith) [reply tests/test.py:274]: `r""` strings may have been designed for regexes, but they're documented as generic unescaped strings with an implementation quirk that you can't end on a trailing `\`.

**Pull request merged**